.templates/openstack/properties.yml

```diff
@@ -0,0 +1,72 @@
+---
+meta:
+  shield:
+    daemon_public_key: (( vault "secret/dc01/proto/shield/keys/core:public" ))
+
+properties:
+  prometheus:
+    rule_files: 
```

Comment:
If you enable the `rule_files` by default, then you need to add the `bosh_alerts` and `cloudfoundry_alerts` to the job templates, otherwise the prometheus will fail because those alerts doesn't exists.